hw/bitstream/BUILD

```diff
@@ -57,32 +57,17 @@ config_setting(
 )
 
 filegroup(
-    name = "test_rom",
+    name = "bitstream",
     testonly = True,
     srcs = select({
         "bitstream_skip": ["skip.bit"],
         "bitstream_vivado": ["//hw/bitstream/vivado:fpga_cw310_test_rom"],
-        "bitstream_gcp_splice": [":gcp_spliced_test_rom"],
-        "//conditions:default": [":gcp_spliced_test_rom"],
+        "//conditions:default": ["@bitstreams//:chip_earlgrey_cw310_bitstream"],
```

Comment:
This is just a stylistic comment and something I'd defer to others on (like @cfrantz ), but I think we'd still prefer a named entry for all options, instead of having one only be selectable with `//conditions:default`. I forget how that might be helpful, though.

Reply:
The code is not consistent on that point, some selects only use default, some don't.

Reply:
I have pushed an update that consistently add a `bitstream_gcp_splice` in every select.